lib/event-listeners/food-listener.js

```diff
@@ -3,10 +3,12 @@ import { createFood } from '../ajax-requests/add_food.js'
 
 const $ = require('jquery')
 
-let foodObj = $("#add-food").on("click", function(event) {
+let foodObj = $("#food_form").on("submit", function(event) {
   event.preventDefault()
   let name = $("#name").val()
   let calories =  $("#calories").val()
   let newFood = new Food(name, calories)
```

Comment:
I would still validate here that the values aren't blank for best practices. Technically the user could edit the html in inspector and get through to the submit.